content/publication_resources/bibtex/lahti.bib

```diff
@@ -12,10 +12,61 @@ @Article{
   annote =	 {}
 }
 
-    
 
 
-    
+@Article{McDonald2023,
+  author =	 {Daniel McDonald, Yueyu Jiang, Metin Balaban, Kalen Cantrell,
+            Qiyun Zhu, Antonio Gonzalez, James T. Morton, Giorgia Nicolaou, 
+            Donovan H. Parks, Søren M. Karst, Mads Albertsen, Philip Hugenholtz, 
+                Todd DeSantis, Se Jin Song, Andrew Bartko, Aki S. Havulinna,
+              Pekka Jousilahti, Susan Cheng, Michael Inouye, Teemu Niiranen, 
+          Mohit Jain, Veikko Salomaa, Leo Lahti, Siavash Mirarab and Rob Knight},
+  title =	 {Greengenes2 unifies microbial data in a single reference tree},
+  journal =	 {Nature Biotechnology},
+  year =	 2023,
+  doi =		 {10.1038/s41587-023-01845-1},
+  keywords =	 {bioscience}
+}    
+
+
+@Article{Gao2023,
+  author =	 {Yu Gao, Yağmur Şimşek, Emma Gheysen, Tuomas Borman, Yi Li, 
+                Leo Lahti, Karoline Faust, Daniel Rios Garza},
+  title =	 {miaSim: an R/Bioconductor package to easily simulate microbial community dynamics},
```

Comment:
this contains intentionally capitalized letters and the entire title should be put in double curly brackets "{{...}}" this will ensure that the capitals will be kept as capitlas in rendering